server/test/unit/server/tasks/test_repository.py

```diff
@@ -166,9 +166,9 @@ def test_distributor_update_with_agent_errors(self, mock_dist_manager, mock_bind
 class TestRepositoryPublish(PulpCeleryTaskTests):
     @patch('pulp.server.managers.repo.publish.RepoPublishManager.queue_publish')
     def test_pass_through_to_manager(self, mock_queue_publish):
-        result = repository.publish('foo', 'dist1', {})
+        result = repository.publish('foo', 'dist1', {}, None)
         # make sure the args get passed through
-        mock_queue_publish.assert_called_once_with('foo', 'dist1', {})
+        mock_queue_publish.assert_called_once_with('foo', 'dist1', {}, None)
```

Comment:
Consider using something other than None for the arg you add. I suggest a string.